package.json

```diff
@@ -9,28 +9,27 @@
   "author": "Mark Erikson <mark@isquaredsoftware.com>",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-typescript": "^7.1.0",
-    "@types/jest": "^23.3.12",
-    "@types/node": "^10.12.18",
+    "@babel/core": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/preset-typescript": "^7.3.3",
+    "@types/jest": "^23.3.14",
+    "@types/node": "^10.14.4",
     "@types/redux-immutable-state-invariant": "^2.1.0",
-    "babel-core": "7.0.0-bridge.0",
+    "@typescript-eslint/parser": "^1.6.0",
     "babel-eslint": "^10.0.1",
-    "eslint": "^4.17.0",
+    "eslint": "^5.16.0",
```

Comment:
The major bump was safe for consumers of the CLI, only API users really needed to worry.